samples/ballerina-by-example/examples/structs/structs.sh

```diff
@@ -1,9 +1,8 @@
 $ ballerina run structs.bal
 {name:"", age:-1, parent:null, status:""}
-{name:"Jack", age:20,
-        parent:{name:"", age:-1, parent:null, status:""},
-        status:""}
+{name:"Jack", age:20, parent:{name:"", age:-1, parent:null, status:""}, status:""}
 Jack
 Jack
 -1
-{name:"Peter", age:25, parent:null, status:""}
\ No newline at end of file
+{name:"Peter", age:25, parent:null, status:""}
+{name:"Jack", age:20, parent:{name:"Peter", age:25, parent:null, status:""}, status:""}
```

Comment:
Checked with the generated UI and currently it has width of 90 which is fine with the change.